doc/command_syntax_usage.rst

```diff
@@ -38,9 +38,9 @@ over SSH. Each authentication identity requires its own auth entry.
 
 *Note:* --password not being passed or passed as empty are considered the same thing.
 
----------------------------
-Password-protected SSH key
----------------------------
+-------------------------
+SSH Key with a Passphrase
+-------------------------
 
 There are two ways to handle password-protected ssh keys. If a
```

Comment:
Sorry to nit pick but I would also think we would use the same language in the paragraph as well as the change to the section name.

Reply:
Sorry, just saw this comment after merging. I will fix the paragraph too.